test/common_nn.py

```diff
@@ -603,12 +603,6 @@ def _analytical_jacobian(self, module, input, jacobian_input=True, jacobian_para
         return res
 
     def _numerical_jacobian(self, module, input, jacobian_input=True, jacobian_parameters=True):
-        output = self._forward(module, input)
-        output_size = output.nelement()
-
-        if jacobian_parameters:
-            param, d_param = self._get_parameters(module)
```

Comment:
This is just a code cleanup. No important fixes here, right?

Reply:
Yes